test/helpers/DeBruijn.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GNU AGPLv3
+pragma solidity ^0.8.0;
+
+library DeBruijn {
+    function isDeBruijnSequence(uint256 seq) public pure returns (bool) {
```

Comment:
I would advise against using public library functions in general since the meaning of external is very different and often unexpected for libraries. This should only require internal.